policies/coding-style.md

```diff
@@ -617,10 +617,11 @@ or have important caveats:
 - Use of `short` or `unsigned short` is deprecated. In the worst case, `short` has
   the same size as `int`.
 
-- Use `int` or `unsigned int` only where the range of values required is a subset
-  of the range `[-32767, 32767]`, as this is guaranteed by the C standard.
+- `int` and `unsigned int` can be used.
 
-  Consider using `int32_t` or `uint32_t` instead.
+  The C standard only guarantees that `int` can represent the range
+  `[-32767, 32767]`, but it is an assumption of the OpenSSL codebase that `int`
+  is 32 bits.
```

Comment:
I'd say `at least 32 bits long`.  `int32_t` or `uint32_t` should be used only in places where the value must be exactly 32 bit long.